uint8 activateRobot
uint8 humanIsReady

